### ISSUE TITLE
Card menu

### ISSUE THREAD

**Issue by JohnONolan** [TryGhost/Ghost-Editor#16]
The menu which is triggered by the slash command (I'm just going to call it the "card menu"), should also be able to be triggered by a `+` icon, which appears here:

![image](https://cloud.githubusercontent.com/assets/120485/23252756/92fcaebe-f9e4-11e6-8ba8-87ab643d5b34.png)

but *only* on a new/empty line where the cursor currently is.

This should replace the in-place-toolbar which is currently there

**Comment by JohnONolan** [comment]
Also, right-clicking while the card-menu is open crashes the editor for me

![image](https://cloud.githubusercontent.com/assets/120485/23253002/790da188-f9e5-11e6-8656-1f7cc743eb6c.png)

**Comment by disordinary** [comment]
Yeah, there are a few scenarios where mobiledoc itself crashes when it looses the range. I need to debug mobiledoc itself unfortunately.

**Comment by disordinary** [comment]
So to be clear we get rid of the block change menu and only have an empty block menu?

One of the issues that I have with Medium is that you can't change your mind on a block type easily.

For instance, if I create an unordered list I have no way to change it to a numbered list without manually going through each item, hitting backspace, then making it a list again.

**Comment by JohnONolan** [comment]
Yes, to start with - purely because the usecase of adding a card is more common than changing one, and changing one card to another card in all but the most simple scenarios (like lists) I think presents a myriad of edge-cases that I'd rather not even think about initially 😬
